src/pykeen/training/training_loop.py

```diff
@@ -637,7 +637,9 @@ def _forward_pass(self, batch, start, stop, current_batch_size, label_smoothing,
         current_epoch_loss = loss.item()
 
         # reset the regularizer to free the computational graph
-        self.model.regularizer.reset()
+        _regularizer = getattr(self.model, 'regularizer', None)
```

Comment:
This is a little janky. It's necessary for the current implementation that includes regularizers, but with the #107 update it needs to be optional

Reply:
other way of writing this could be:
```python
try:
    self.regularizer.reset()
except AttributeError:
    pass
```

Reply:
One more:

```python
if hasattr(self.model, "regularizer"): # ...
```

Reply:
that one manages to have the fewest lines, I think I'll take it :)

Reply:
Resolved in cc49f6aa